incidents/ugent_20231003_tier1.yml

```diff
@@ -1,7 +1,7 @@
 meta_data:
-  title: Tier-1 workernodes down due to problems with cooling
+  title: (RESOLVED) Tier-1 workernodes down due to problems with cooling
   start_date: 2023-10-03 15:10:00
-  end_date: 
+  end_date: 2023-10-06 18:00:00
```

Comment:
we started jobs at 1pm.

Reply:
ah, I see, you use this to keep the page online even after the incident was already done.

Reply:
I understand, but if I set the end time to 13:00, then the entry totally disappears on status page...

So I set a later end time, and add `RESOLVED` to the incident title, and mention exact time of fix in body of the incident.